direnvrc

```diff
@@ -129,7 +129,7 @@ use_flake() {
     local tmp_profile_rc
     tmp_profile_rc=$("${NIX_BIN_PREFIX}nix" print-dev-env \
       --extra-experimental-features "nix-command flakes" \
-      --profile "$tmp_profile" "$flake_expr")
+      --profile "$tmp_profile" $@)
```

Comment:
I think this should have the desired effect too and handle quoting properly:

```suggestion
      --profile "$tmp_profile" "$@")
```

Reply:
also see shellcheck

Reply:
TIL that POSIX handles quoting `$@` in a way that's both what I wanted all along and pretty inconsistent with how it quotes most other variables...

This should do exactly what I wanted.